### PR TITLE
tidying up

### PR DESCRIPTION
**Pull request by garrettbingham**
Minor changes; see comments in each file!
Pylint is happy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ and ensure your TensorFlow version is at least 2.4.0.
 
 Import the `AutoInit` class in your script
 ```python
-from autoinit.initializer.initialize_weights import AutoInit
+from autoinit import AutoInit
```
Comment (garrettbingham) [inline README.md:25]: Easier import now

File: README.md [hunk 2]
```diff
@@ -62,7 +62,12 @@ Pooling and recurrent layers require Monte Carlo sampling to estimate the outgoi
 
 ## Visualization
 
-Use `autoinit/util/visualize_init.py` to visualize signal propagation across a default and AutoInit version of a given neural network.  The docstring to the `AutoInitVisualizer` class contains more information.
+To plot signal propagation across a default and AutoInit version of a given neural network, use:
+```python
+from autoinit import AutoInitVisualizer
+AutoInitVisualizer().visualize(training_model)
+```
+The docstring to the `AutoInitVisualizer` class in `autoinit/util/visualize_init.py` contains additional information.
```
Comment (garrettbingham) [inline README.md:70]: Concrete example of visualization.  More details still available in docstring.

File: autoinit/util/visualize_init.py [hunk 3]
```diff
@@ -68,7 +68,7 @@ def _get_accessible_outputs(self, model, warn=True):
 
     def _load_models(self, model, custom_objects=None):
         """
-        This function creates a default initialized and universal initialized model.
+        This function creates a default initialized and AutoInit initialized model.
```
Comment (garrettbingham) [inline autoinit/util/visualize_init.py:71]: There were a few instances in this file with the old name.

File: setup.py [hunk 4]
```diff
@@ -12,8 +12,8 @@
 from setuptools import setup, find_packages
 
 setup(name='autoinit',
-    version='0.0.1',
-    description='AutoInit: Analytic Mean- and Variance-Preserving Weight Initialization for Neural Networks',
+    version='0.0.2',
+    description='AutoInit: Analytic Signal-Preserving Weight Initialization for Neural Networks',
```
Comment (garrettbingham) [inline setup.py:16]: We changed the paper name to say signal-preserving

File: autoinit/util/visualize_init.py [hunk 5]
```diff
@@ -87,9 +87,11 @@ def _load_models(self, model, custom_objects=None):
             raise TypeError('The model argument must be a TensorFlow Model, config dictionary, '\
                            f'or JSON string.  Got type {type(model)}.')
 
-        # Create the universal model based on the default model
+        # Create the AutoInit model based on the default model
         self.autoinit_model, self.mean_var_estimates = \
-            AutoInit().initialize_model(self.default_model.get_config(), True)
+            AutoInit().initialize_model(self.default_model.get_config(),
+                                        return_estimates=True,
+                                        custom_objects=custom_objects)
```
Comment (garrettbingham) [inline autoinit/util/visualize_init.py:94]: Need to pass custom objects dictionary for visualization to work with custom models.

**Review by ofrancon — APPROVED** [review]
lgtm!

**Pull request merged**